pages/exercises/[lessonSlug].tsx

```diff
@@ -66,6 +66,7 @@ const Exercises: React.FC<QueryDataProps<GetExercisesQuery>> = ({
     <Layout title={currentLesson.title}>
       {exercise ? (
         <Exercise
+          key={exerciseIndex}
```

Comment:
This line fixes a small React key bug which previously would make it possible to press "Show Answer" (to show the current exercise answer) then "Skip" (to skip to the next exercise) which caused the next exercise's answer to be shown.